srcalc/src/main/java/gov/va/med/srcalc/domain/model/DiscreteTerm.java

```diff
@@ -135,6 +135,6 @@ public int hashCode()
     public String toString()
     {
         return String.format(
-                "%s=%s?->%s", getVariable(), getOption(), getCoefficient());
+                "%s=%s->%s", getVariable(), getOption(), getCoefficient());
```

Comment:
Even though it's not a "Boolean Term", it is still a boolean condition (A == B). This is as opposed to NumericalTerm, which has no boolean aspect to it at all.

It could go both ways, but we shouldn't change code that is already on master without a good reason. Please revert the change.